monai/deploy/operators/monai_bundle_inference_operator.py

```diff
@@ -62,43 +63,110 @@ def get_bundle_config(bundle_path, config_names):
     Gets the configuration parser from the specified Torchscript bundle file path.
     """
 
-    def _read_from_archive(archive, root_name: str, relative_path: str, path_list: List[str]):
-        """A helper function for reading a file in an zip archive.
+    bundle_suffixes = [".json", ".yaml"]  # The only supported file ext(s)
```

Comment:
I would suggest to refer to the MONAI ConfigParser for suffixes:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/bundle/config_parser.py#L381

Thanks.

Reply:
Addressed in the last commit.